src/analysis/convex/function.lean

```diff
@@ -104,6 +104,15 @@ lemma strict_concave_on.subset {t : set E} (hf : strict_concave_on 𝕜 t f) (hs
   strict_concave_on 𝕜 s f :=
 ⟨hs, λ x y hx hy, hf.2 (hst hx) (hst hy)⟩
 
+section composition
+variables [has_smul 𝕜 Q] [ordered_add_comm_monoid Q] (t : set β) (g : β → Q)
```

Comment:
Why do you need `Q`? What's wrong with `F`? Also, do you need `t` and `g` to be explicit?